### DISCUSSION TITLE
Portenta H7 MDNS Library compatibility

### DISCUSSION THREAD

**Discussion by Zuroy** [khoih-prog/MDNS_Generic#3] [Q&A]
Good morning everyone,
I wonder if this library is compatible with Portenta H7 board. I tryed to use it with that board to add (register) a new service on my net but when I lunch the discovery routine using python zeroconf it returns this error: 
![image](https://user-images.githubusercontent.com/45754537/131466125-f3378405-4723-4238-b209-d1358f652655.png)

Can you help me?
Thank you very much,

Amedeo

**Comment by khoih-prog** [comment]
Fixed in [MDNS_Generic releasesv1.3.0](https://github.com/khoih-prog/MDNS_Generic/releases/tag/v1.3.0). 

For more info, check [Portenta H7 MDNS Library compatibility#4](https://github.com/khoih-prog/MDNS_Generic/issues/4)
